auth/server/src/main/java/com/seagates3/parameter/validator/S3ParameterValidatorUtil.java

```diff
@@ -456,4 +456,19 @@ public static Boolean isValidGroupName(String groupName) {
         return !(groupName.length() < 1
                 || groupName.length() > MAX_GROUP_NAME_LENGTH);
     }
+
+   public
+    static Boolean isValidPolicyARN(String arn) {
```

Comment:
Its not already available. 
This will be used for validating policy arn format.
Bucket Policy uses different regex because in bucket policy, many combinations under ARN are possible-
example  - Full USER ARN, Root account ARN, ARN with * at end.